### PR TITLE
[Storage] Update default config.

### PR DESCRIPTION
**Pull request by grao1991**
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2968)

### CODE REVIEW

**Review by JoshLind — APPROVED** [review]
LGTM. I'm assuming @msmouse is happy with this change 😄

File: config/src/config/storage_config.rs [hunk 1]
```diff
@@ -143,7 +143,7 @@ impl Default for LedgerPrunerConfig {
     fn default() -> Self {
         LedgerPrunerConfig {
             enable: true,
-            prune_window: 10_000_000,
+            prune_window: 150_000_000,
```
Comment (sitalkedia) [inline config/src/config/storage_config.rs:146]: nit: Can you add a comment explaining the reasoning/calculation behind these numbers?

**Review by sitalkedia — APPROVED** [review]

File: config/src/config/storage_config.rs (hunk 1 above)
Comment (msmouse) [reply config/src/config/storage_config.rs:146]: +1 I'd like a rationale for this number commented in.

Comment (grao1991) [reply config/src/config/storage_config.rs:148]: Comment added. (tl;dr it's still kind of magic numbers.)

**Pull request merged**